helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/JsonUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Collection;
+import java.util.Map;
+import org.apache.avro.AvroRuntimeException;
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.node.JsonNodeFactory;
+
+
+public class JsonUtils {
+  static final String BYTES_CHARSET = "ISO-8859-1";
+
+  /**
+   * toJsonNode takes in an arbitrary object and convert it into corresponding JsonNode
+   * @param datum input datum object
+   * @return generated JsonNode object.
+   */
+  public static JsonNode toJsonNode(Object datum) {
```

Comment:
this method should probably have Jackson1 in the name, since it returns jackson 1 objects (jackson 2 is under the fasterxml package)